packages/spectacle/src/hooks/use-deck-state.ts

```diff
@@ -49,8 +50,15 @@ function deckReducer(state: DeckState, { type, payload = {} }: ReducerActions) {
         initialized: true
       };
     case 'SKIP_TO':
+      const navigationDirection = (() => {
+        if ('slideIndex' in payload && payload.slideIndex) {
+          return clamp(payload.slideIndex - state.activeView.slideIndex, -1, 1);
+        }
+        return null;
+      })();
```

Comment:
Doing a computed value in an iife since I need to type narrow `slideIndex` on `payload` since this reducer supports partial values.